docs/guide/db-migrations.md

```diff
@@ -152,8 +152,8 @@ to `Schema::TYPE_STRING` to specify that the column cannot be null.
 > Info: The mapping between abstract types and physical types is specified by
   the [[yii\db\QueryBuilder::$typeMap|$typeMap]] property in each concrete `QueryBuilder` class.
 
-Since version 2.0.6 schema builder which provides more convenient way defining column schema was introduced so migration above
-could be written like the following:
+Since version 2.0.6, the schema builder provieds more convenient way of defining column schema.
```

Comment:
provieds → provides

Also overall it's not correct overall. Schema builder is new thing in 2.0.6.